<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="6" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-07-29T10:24:27 IST" name="Suite" finished-at="2025-07-29T10:24:54 IST" duration-ms="26919">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidations.LoginErrorValidation()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" name="LoginErrorValidation" class="princepinnu.secondSeleniumFramework.ErrorValidations"/>
      </group> <!-- ErrorHandling -->
      <group name="Purchase">
        <method signature="SubmitOrderTest.submitOrder(java.util.HashMap)[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" name="submitOrder" class="princepinnu.secondSeleniumFramework.SubmitOrderTest"/>
      </group> <!-- Purchase -->
      <group name="ProductErrorValidation">
        <method signature="ErrorValidations.ProductErrorValidation()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" name="ProductErrorValidation" class="princepinnu.secondSeleniumFramework.ErrorValidations"/>
      </group> <!-- ProductErrorValidation -->
    </groups>
    <test started-at="2025-07-29T10:24:27 IST" name="Error Validations Test" finished-at="2025-07-29T10:24:51 IST" duration-ms="24131">
      <class name="princepinnu.secondSeleniumFramework.ErrorValidations">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" started-at="2025-07-29T10:24:27 IST" name="launchApplication" finished-at="2025-07-29T10:24:34 IST" duration-ms="7274" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" started-at="2025-07-29T10:24:34 IST" name="LoginErrorValidation" finished-at="2025-07-29T10:24:35 IST" duration-ms="888" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at princepinnu.secondSeleniumFramework.ErrorValidations.LoginErrorValidation(ErrorValidations.java:22)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" started-at="2025-07-29T10:24:35 IST" name="tearDown" finished-at="2025-07-29T10:24:36 IST" duration-ms="379" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" started-at="2025-07-29T10:24:36 IST" name="launchApplication" finished-at="2025-07-29T10:24:41 IST" duration-ms="5479" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" started-at="2025-07-29T10:24:41 IST" name="LoginErrorValidation" finished-at="2025-07-29T10:24:42 IST" duration-ms="842" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at princepinnu.secondSeleniumFramework.ErrorValidations.LoginErrorValidation(ErrorValidations.java:22)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" started-at="2025-07-29T10:24:43 IST" name="tearDown" finished-at="2025-07-29T10:24:43 IST" duration-ms="258" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" started-at="2025-07-29T10:24:43 IST" name="launchApplication" finished-at="2025-07-29T10:24:48 IST" duration-ms="5232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" started-at="2025-07-29T10:24:48 IST" name="ProductErrorValidation" finished-at="2025-07-29T10:24:51 IST" duration-ms="2637" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:princepinnu.secondSeleniumFramework.ErrorValidations@295cf707]" started-at="2025-07-29T10:24:51 IST" name="tearDown" finished-at="2025-07-29T10:24:51 IST" duration-ms="211" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- princepinnu.secondSeleniumFramework.ErrorValidations -->
    </test> <!-- Error Validations Test -->
    <test started-at="2025-07-29T10:24:27 IST" name="Submit Order Test" finished-at="2025-07-29T10:24:54 IST" duration-ms="26919">
      <class name="princepinnu.secondSeleniumFramework.SubmitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" started-at="2025-07-29T10:24:27 IST" name="launchApplication" finished-at="2025-07-29T10:24:33 IST" duration-ms="5188" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" started-at="2025-07-29T10:24:33 IST" name="submitOrder" data-provider="getData" finished-at="2025-07-29T10:24:37 IST" duration-ms="4589" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Prince@2000, product=ZARA COAT 3, email=princebabu@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" started-at="2025-07-29T10:24:37 IST" name="tearDown" finished-at="2025-07-29T10:24:38 IST" duration-ms="306" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" started-at="2025-07-29T10:24:38 IST" name="launchApplication" finished-at="2025-07-29T10:24:43 IST" duration-ms="4956" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" started-at="2025-07-29T10:24:43 IST" name="submitOrder" data-provider="getData" finished-at="2025-07-29T10:24:47 IST" duration-ms="4554" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Tharun@1234, product=ADIDAS ORIGINAL, email=thatunbabu@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" started-at="2025-07-29T10:24:47 IST" name="tearDown" finished-at="2025-07-29T10:24:47 IST" duration-ms="342" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" started-at="2025-07-29T10:24:47 IST" name="launchApplication" finished-at="2025-07-29T10:24:53 IST" duration-ms="5307" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="OrderHistoryTest()[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" started-at="2025-07-29T10:24:53 IST" name="OrderHistoryTest" finished-at="2025-07-29T10:24:54 IST" depends-on-methods="princepinnu.secondSeleniumFramework.SubmitOrderTest.submitOrder" duration-ms="1036" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:princepinnu.secondSeleniumFramework.SubmitOrderTest@7ac296f6]" started-at="2025-07-29T10:24:54 IST" name="tearDown" finished-at="2025-07-29T10:24:54 IST" duration-ms="212" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- princepinnu.secondSeleniumFramework.SubmitOrderTest -->
    </test> <!-- Submit Order Test -->
  </suite> <!-- Suite -->
</testng-results>
